src/getExports.js

```diff
@@ -4,12 +4,21 @@ import { parse } from '@babel/parser';
 import { toRelativePath } from './utils';
 
 export default function getExports(sourcePaths, ctx) {
-  return sourcePaths.map(sourcePath => {
+  const result = [];
+
+  for (const sourcePath of sourcePaths) {
```

Comment:
Explicitly went for `for` loop to avoid double looping through array with `filter` and `map`